conf/prod.routes

```diff
@@ -3,6 +3,5 @@
 ->         /account              app.Routes
 ->         /                     health.Routes
 ->         /template             template.Routes
-->         /internal             internal.Routes
```

Comment:
Just one comment - should we not also removed the internal.Routes file from the conf folder?